addons/sale_stock/models/sale_order.py

```diff
@@ -428,6 +428,9 @@ def write(self, values):
         if 'product_uom_qty' in values:
             lines = self.filtered(lambda r: r.state == 'sale' and not r.is_expense)
 
+        if 'product_packaging_id' in values:
+            self.move_ids.group_id.stock_move_ids.product_packaging_id = values['product_packaging_id']
```

Comment:
It will takes too much moves. You should rather do an override in stock_move and if you write on product_packaging_id you should propagate it to it's move_orig_ids

Reply:
Just write it on self.move_ids here.